### PR TITLE
Update date data type casting

### PR DESCRIPTION
**Pull request by fivetran-reneeli**
## PR Overview
**This PR will address the following Issue/Feature:** https://github.com/fivetran/dbt_quickbooks/issues/96

**This PR will result in the following new package version:**

0.11.2

Non breaking as these changes are under the hood and will not have visible impact on downstream model materializations.
This will also be in combo with [PR #51](https://github.com/fivetran/dbt_quickbooks_source/pull/51) in the source

**Please detail what change(s) this PR introduces and any additional information that should be known during the review of this PR:** 

- Update seeds and configs in the integration testing folder to match what was updated in the [source]( [PR #51](https://github.com/fivetran/dbt_quickbooks_source/pull/51))

### Basic Validation
Please acknowledge that you have successfully performed the following commands locally:
- [x] dbt compile
- [x] dbt run –full-refresh
- [x] dbt run
- [x] dbt test
- [x] dbt run –vars (if applicable)

Before marking this PR as "ready for review" the following have been applied:
- [x] The appropriate issue has been linked and tagged
- [x] You are assigned to the corresponding issue and this PR
- [x] BuildKite integration tests are passing

### Detailed Validation
Please acknowledge that the following validation checks have been performed prior to marking this PR as "ready for review":
- [x] You have validated these changes and assure this PR will address the respective Issue/Feature.
- [x] You are reasonably confident these changes will not impact any other components of this package or any dependent packages.
- [x] You have provided details below around the validation steps performed to gain confidence in these changes.

Models should successfully run. Additionally will ask the customer to run on this branch as well.

### Standard Updates
Please acknowledge that your PR contains the following standard updates:
- Package versioning has been appropriately indexed in the following locations:
    - [x] indexed within dbt_project.yml
    - [x] indexed within integration_tests/dbt_project.yml
- [x] CHANGELOG has individual entries for each respective change in this PR
    
- [n/a] README updates have been applied (if applicable)
    
- [ n/a] DECISIONLOG updates have been updated (if applicable)
- [ n/a] Appropriate yml documentation has been added (if applicable)

### dbt Docs
Please acknowledge that after the above were all completed the below were applied to your branch:
- [ ] docs were regenerated (unless this PR does not include any code or yml updates)

### If you had to summarize this PR in an emoji, which would it be?
 
:dancer:

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# dbt_quickbooks v0.11.2
+This PR includes the following updates:
```
Comment (fivetran-avinash) [inline CHANGELOG.md:2]:
```suggestion
[This PR](https://github.com/fivetran/dbt_quickbooks/pull/103/files) includes the following updates:
```

**Review by fivetran-avinash — APPROVED** [review]
looks good with one small commit update! Make sure to revert `packages.yml` and test that buildkite passes before merging/releasing.

**Comment by fivetran-reneeli** [comment]
reminder to change package deps before merging

**Review by fivetran-avinash — COMMENTED** [review]
@fivetran-reneeli Re-reviewed! Left a few comments.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+# dbt_quickbooks v0.11.2
```
[the remark this answers is not in the document]
Comment (fivetran-reneeli) [reply CHANGELOG.md:1]: oh good point, it'll serve as a good reminder as well. updated!

File: dbt_project.yml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 config-version: 2
 name: 'quickbooks'
 
-version: '0.11.1'
+version: '0.11.2'
```
[the remark this answers is not in the document]
Comment (fivetran-reneeli) [reply dbt_project.yml:4]: good point

File: integration_tests/dbt_project.yml [hunk 4]
```diff
@@ -1,6 +1,6 @@
 name: 'quickbooks_integration_tests'
 
-version: '0.11.1'
+version: '0.11.2'
```
[the remark this answers is not in the document]
Comment (fivetran-reneeli) [reply integration_tests/dbt_project.yml:3]: good point

File: integration_tests/seeds/bill_payment_line_data.csv [hunk 5]
```diff
@@ -1,5 +1,5 @@
 bill_payment_id,index,_fivetran_synced,amount,bill_id,check_id,credit_card_charge_id,credit_card_credit_id,deposit_id,expense_id,journal_entry_id,linked_bill_payment_id,vendor_credit_id,bill_payment_credit_card_id
-23,0,2020-05-20 14:03:30,38757,19,,,,,,,,,
+23,0,2020-05-20 14:03:30,38757,19.25,,,,,,,,,
```
[the remark this answers is not in the document]
Comment (fivetran-reneeli) [reply integration_tests/seeds/bill_payment_line_data.csv:2]: Ooh good catch, I'm not sure why that's there. I'll change those to 19

**Comment by fivetran-avinash** [comment]
@fivetran-reneeli New changes look good to me!

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,7 @@
+# dbt_quickbooks v0.VERSION.VERSION
+[This PR](https://github.com/fivetran/dbt_quickbooks/pull/103/files) includes the following updates:
```
Comment (fivetran-jamie) [inline CHANGELOG.md:2]: should it be 
```suggestion
[PR #103](https://github.com/fivetran/dbt_quickbooks/pull/103/files) includes the following updates:
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,3 +1,7 @@
+# dbt_quickbooks v0.VERSION.VERSION
```
Comment (fivetran-jamie) [inline CHANGELOG.md:1]: if we are updating the packages.yml we gotta cut a release for the transform

File: CHANGELOG.md (hunk 6 above)
Comment (fivetran-reneeli) [reply CHANGELOG.md]: updated!

File: CHANGELOG.md (hunk 7 above)
Comment (fivetran-reneeli) [reply CHANGELOG.md]: True, updated this

**Pull request merged**